### PR TITLE
add Scaleway to the list of knative offerings

### PR DESCRIPTION
**Pull request by sifrah**
- Adds Scaleway Serverless to the list of knative-offerings

### CODE REVIEW

**Comment by sifrah** [comment]
@googlebot I signed it!

**Comment by dprotaso** [comment]
/unassign @dprotaso 

bumping @rgregg (for steering approval)

**Comment by rgregg** [comment]
Before approving, the link should point to somewhere specific about the Serverless / Knative-based offering. Right now the link lands on Scaleway's homepage, which doesn't have any indication about Knative or Serverless. I believe all the other links are to specific landing pages for Serverless functionality.

If the link is updated to point to a specific page that highlights the Serverless offering and Knative, then I'd approve.

File: docs/knative-offerings.md [hunk 1]
```diff
@@ -19,8 +19,10 @@ Here is a list of commercial Knative products (alphabetically):
...
 - [Openshift Serverless](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/html-single/serverless_applications/index): on OpenShift Container Platform enables stateful, stateless, and serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
 - [Pivotal Function Service (PFS)](https://docs.pivotal.io/pfs): is a platform for building and running functions, applications, and containers on Kubernetes. PFS is based on the [riff](https://projectriff.io) open source project.
+- [Scaleway Serverless](https://www.scaleway.com/en/): The serverless platform from the Scaleway cloud provider allows you to put your functions and containers in production easily without having to manage a Kubernetes cluster.
```
Comment (rgregg) [inline docs/knative-offerings.md:22]: Please point to a specific topic about serverless, not the Scaleway home page.

**Comment by abrennan89** [comment]
@sifrah any updates on this? Is there a Serverless specific link in Scaleway docs this can point to?

**Comment by sifrah** [comment]
Hi @abrennan89, we currently working on this, I will ping you when it will be done

**Comment by abrennan89** [comment]
@sifrah any further updates on this? If not, can this PR be closed until there's something to add?

**Comment by sifrah** [comment]
Hi @abrennan89 we don't have any landing page at this time :'( 
So we can close this PR and will probably open another one when we will be ok 
Thanks,

**Comment by mattmoor** [comment]
@sifrah It might also be interesting to consider talking about how you are using Knative on the blog, assuming it can fit within the blog guidelines.  cc @macruzbar

**Pull request closed without merging**